pkg/bgpv1/manager/reconciler/neighbor.go

```diff
@@ -104,19 +107,19 @@ func (r *NeighborReconciler) Reconcile(ctx context.Context, p ReconcileParams) e
 	}
 
 	// populate set from universe of current neighbors
-	for i, n := range curNeigh {
+	for _, n := range curNeigh {
 		var (
-			key = r.neighborID(&n)
+			key = r.neighborID(n)
 			h   *member
 			ok  bool
 		)
 		if h, ok = nset[key]; !ok {
 			nset[key] = &member{
-				cur: &curNeigh[i],
+				cur: n,
```

Comment:
Is range issue resolved in latest go versions ? I think in latest go versions `n` is new on each iteration of range. Previously it was not the case so pointer to exact element in the slice was referenced.

Reply:
Now `curNeigh` is a pointer slice, so I believe it's safe to copy `n` here.